meta-jsdelivr/recipes-jsdelivr/jsdelivr-container-manager/files/jsdelivr-container-loader.sh

```
@@ -0,0 +1,399 @@
+# Log function
+log() {
+    echo "[Container Loader] $1" > /dev/tty3
+    echo "[Container Loader] $1"
+}
+
+# Load configuration from persistent storage if available, otherwise use default
+load_config() {
+    # Check if persistent config exists
+    if [ -f "$PERSIST_CONFIG_FILE" ]; then
+        log "Loading configuration from persistent storage"
+        source "$PERSIST_CONFIG_FILE"
```

Shellcheck (0.11.0)

[warning] ShellCheck can't follow non-constant source. Use a directive to specify location.

(SC1090)